inference-engine/tests/functional/plugin/shared/src/subgraph_tests/multioutput_eltwise_squeeze_eltwise.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include <debug.h>
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/precision_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "subgraph_tests/multioutput_eltwise_squeeze_eltwise.hpp"
+
+namespace LayerTestsDefinitions {
```

Comment:
current code as i remember lacks of correct handling in certain cases, if you didt add case that break it, worth creating different test. Please investigate that more deeply